feat: Added 'Secret Exports' schema for Bicep (#1261)
Azure/Azure-Verified-Modules: docs/content/specs-defs/specs/bicep/interfaces.md

---
title: Interfaces
geekdocNav: true
geekdocAlign: left
geekdocAnchor: true
url: /specs/bicep/interfaces/
---

> Below are the interfaces/schemas for AVM Resource Modules features/extension resources only relevant to Bicep.

Reviewer: ```suggestion
This page details the Bicep specific interfaces/schemas for AVM Resource Modules features/extension resources.
```

Author: I guess it should be `Bicep-specific`

{{< toc >}}

<br>

## Secrets export

The export of secrets from inside a module to a provided Key Vault reference is something that can be implement as per demand following the below schema.

> It's implementation provides a secure way around Bicep's current limitation to be unable to securely output secrets.

Author: How about
> _This implementation provides a secure way around the current limitation of Bicep, which is to provide a secure template output_

> It's implementation provides a secure way around Bicep's current limitation to be unable to securely output secrets.

Reviewer: I changed it to something very close :)
